src/Sulu/Bundle/MediaBundle/Media/Exception/MediaException.php

```diff
@@ -138,6 +138,12 @@ class MediaException extends Exception
      */
     const EXCEPTION_FILENAME_ALREADY_EXISTS = 5020;
 
+    /**
+     * @var int
+     * @description this exception code is thrown when the file is not found
```

Comment:
`@description` is redundant, and not part of phpdoc syntax: http://www.phpdoc.org/docs/latest/index.html

````
/**
 * This exception code is thrown when the file is not found
 * @var int
 */
````